_includes/header.html

```diff
@@ -1,6 +1,42 @@
 <div id="skip-link">
   <a href="#main" class="element-invisible element-focusable">Skip to main content</a>
 </div>
+<header id="header-esteemed" class="global-header" role="header">
+  <div class="container">
+    <div class="global-header-body">
+      <div class="siteHeader-body-left">
```

Comment:
Instead of doing custom CSS for alignment, I'd rather see you using Bootstrap's container/row/column strategy.